### ISSUE TITLE
Detect missing library dependency in manifest.json based on controls used in XML views/fragments

### ISSUE THREAD

**Issue by flovogt** [SAP/ui5-linter#244]
# Scenario
I have an `sap/ui/core/mvc/XMLView` or an in XML defined `sap/ui/core/Fragment`. I am using a control e.g. `sap/m/Text` but I have forgotten to add the `sap.m` library to the `sap.ui5/dependencies/lib` array in the `manifest.json`.

## View
```xml
<mvc:View xmlns:mvc="sap.ui.core.mvc" xmlns:core="sap.ui.core" xmlns="sap.m"controllerName="ui5app.controller.Home" >
  <Text text="Hello"/>
</mvc:View>
```

## manifest.json
```json
"sap.ui5": {
  "dependencies": {
      "minUI5Version": "1.121.0",
      "libs": {
        "sap.ui.core": {}
      }
}
```

# Goal
This scenario is detectedn and an finding is reported to add the missing library dependency to the `manifest.json`.

**Comment by flovogt** [comment]
This issue will be tackled in backlog item CPOUI5FOUNDATION-1083.
